client/verta/verta/client.py

```diff
@@ -671,7 +671,12 @@ def get_or_create_registered_model(self, name=None, desc=None, labels=None, work
         id : str, optional
             ID of the registered_model. This parameter cannot be provided alongside `name`, and other
             parameters will be ignored.
-
```

Comment:
Please keep this empty line before

```
Returns
--------
```

below.

Reply:
done